lib/core/client.js

```diff
@@ -909,7 +909,10 @@ function _resume (client, sync) {
       request.body = null
     }
 
-    if (client.running && util.isStream(request.body)) {
+    if (client.running &&
+      !util.isBuffer(request.body) &&
+      typeof request.body !== 'string' &&
```

Comment:
I don't think body can be a string here?

Reply:
It failed without these checks. I'll have to look back on it though

Reply:
Let me know what tests fail and I can maybe give a hint.

Reply:
This test fails when I remove the buff and string checks: https://github.com/nodejs/undici/blob/master/test/client-pipelining.js#L453